src/mono/wasm/debugger/DebuggerTestSuite/CallFunctionOnTests.cs

```diff
@@ -85,42 +85,36 @@ public async Task CheckVSCodeTestFunction2(string eval_fn, string bp_loc, int li
                     new { @value = num_elems_fetch }
                 }),
                 test_fn: async (result) =>
-               {
-
-                   var is_js = bp_loc.EndsWith(".js", StringComparison.Ordinal);
-
-                   // isOwn = false, accessorPropertiesOnly = true
-                   var obj_accessors = await cli.SendCommand("Runtime.getProperties", JObject.FromObject(new
-                   {
-                       objectId = result.Value["result"]["objectId"].Value<string>(),
-                       accessorPropertiesOnly = true,
-                       ownProperties = false
-                   }), token);
-                   if (is_js)
-                       await CheckProps(obj_accessors.Value["result"], new { __proto__ = TIgnore() }, "obj_accessors");
-                   else
-                       AssertEqual(0, obj_accessors.Value["result"]?.Count(), "obj_accessors-count");
-
-                   // Ignoring the __proto__ property
-
-                   // isOwn = true, accessorPropertiesOnly = false
-                   var obj_own = await cli.SendCommand("Runtime.getProperties", JObject.FromObject(new
-                   {
-                       objectId = result.Value["result"]["objectId"].Value<string>(),
-                       accessorPropertiesOnly = false,
-                       ownProperties = true
-                   }), token);
-
-                   var obj_own_val = obj_own.Value["result"];
-                   var num_elems_recd = len == 0 ? 0 : num_elems_fetch;
-                   AssertEqual(is_js ? num_elems_recd + 1 : num_elems_recd, obj_own_val.Count(), $"obj_own-count");
-
-                   if (is_js)
-                       CheckObject(obj_own_val, "__proto__", "Object");
+                {
 
-                   for (int i = fetch_start_idx; i < fetch_start_idx + num_elems_recd; i++)
-                       CheckNumber(obj_own_val, i.ToString(), 1000 + i);
-               });
+                    var is_js = bp_loc.EndsWith(".js", StringComparison.Ordinal);
+
+                    // isOwn = false, accessorPropertiesOnly = true
+                    var obj_accessors = await cli.SendCommand("Runtime.getProperties", JObject.FromObject(new
+                    {
+                        objectId = result.Value["result"]["objectId"].Value<string>(),
+                        accessorPropertiesOnly = true,
+                        ownProperties = false
+                    }), token);
+
+                    if (!is_js)
```

Comment:
What does `obj_accessors.Value["result"]` have for js?

Reply:
I can keep this check! :) Fixing it!

[
  {
    "name": "__proto__",
    "get": {
      "type": "function",
      "className": "Function",
      "description": "function get __proto__() { [native code] }",
      "objectId": "5950540143256936376.2.13"
    },
    "set": {
      "type": "function",
      "className": "Function",
      "description": "function set __proto__() { [native code] }",
      "objectId": "5950540143256936376.2.14"
    },
    "configurable": true,
    "enumerable": false,
    "isOwn": false
  }
]